pandas/core/arrays/interval.py

```diff
@@ -269,23 +272,39 @@ def _simple_new(
         cls: type[IntervalArrayT],
         left,
         right,
-        closed: IntervalClosedType | None = None,
-        copy: bool = False,
-        dtype: Dtype | None = None,
+        dtype: IntervalDtype,
         verify_integrity: bool = True,
```

Comment:
ideally we'd get verify_integrity out of _simple_new

Reply:
I've made an implementation for this, see latest commit.